samples/Http2SampleApp/cert/testCert.key

```diff
@@ -0,0 +1,51 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Assuming that TlsStream cannot be modified to accept testCert.pfx, I would just throw this in `test/shared/TestCertificates`.

Reply:
Sure. It may be possible to modify it but since a) it's temporary and b) I don't know how to OpenSSL very well, I don't feel like investing time to do it :).

Reply:
@anurse hold my beer ;)
#2150

Reply:
Hey, if you want to invest the time into our never-gonna-ship code, I'm not going to complain :)